astropy_timeseries/io/core.py

```diff
@@ -0,0 +1,109 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import warnings
+
+import numpy as np
+
+from astropy.io import registry, fits
+from astropy.table import Table
+from astropy.time import Time, TimeDelta
+
+from astropy_timeseries.sampled import TimeSeries
+
+__all__ = ["unified_reader"]
+
+
+def _unsupported(hdulist):
```

Comment:
I don't understand this function name - also this doesn't return True/False but an HDU so the docstring is not quite accurate.

Reply:
The idea was that we would add Instrument checks to this function so it would check if a HDU list was supported. I will remove this.